src/main/java/com/sfl/cronofy/api/model/request/FreeBusyRequest.java

```diff
@@ -144,8 +144,12 @@ public void setIncludeManaged(final Boolean includeManaged) {
         this.includeManaged = includeManaged;
     }
 
-    public List<String> getCalendarIds() {
-        return calendarIds;
+    public Object[] getCalendarIds() {
+        if (calendarIds == null) {
```

Comment:
Remembering why I did it now, there's a tight coupling between the tests and the implementation as it's checking every call to the builder rather than the URI generated at the end and I didn't know how to break that without changing a lot of stuff.